source/base/data_out_base.cc

```diff
@@ -7079,11 +7079,11 @@ DataOutInterface<dim, spacedim>::write_vtu_with_pvtu_record(
           filename_vector.emplace_back(filename);
         }
 
-      std::ofstream master_output((directory + filename_master).c_str());
-      this->write_pvtu_record(master_output, filename_vector);
+      std::ofstream main_output((directory + filename_parent).c_str());
```

Comment:
again, you are mixing "main" and "parent" here. ``filename_master`` and ``master_output`` refer to the same thing.

Reply:
Much later on I changed the tutorial programs to use `pvtu_output` and `pvtu_filename`. I've now done that here to be consistent.